creativecloud/blocks/catalog/catalog.js

```diff
@@ -6,6 +6,57 @@ const { loadStyle } = await import(`${miloLibs}/utils/utils.js`);
 // Helps with TBT: MWPW-145127
 loadStyle(`${miloLibs}/blocks/global-navigation/features/profile/dropdown.css`);
 
+function handleCustomAnalyticsEvent(eventName, element) {
```

Comment:
Should we maybe add some tests for `handleCustomAnalyticsEvent` and `enableAnalytics`?